torch/utils/data/datapipes/iter/grouping.py

```diff
@@ -23,67 +22,6 @@ class SHARDING_PRIORITIES(IntEnum):
     MULTIPROCESSING = 3
 
 
-@functional_datapipe('sharding_filter')
```

Comment:
Let's move `SHARDING_PRIORITIES` as well. And, if we do so, we need to land the changes for torchdata.

Reply:
> And, if we do so, we need to land the changes for torchdata.

What kind of changes? -- Change the import right? . What's the procedure of doing that usually? ~

Reply:
> Change the import right?

Yeah

> What's the procedure of doing that usually? 

Open a Diff that contains changes for both pytorch and torchdata. Then, land this diff directly. The CI for TorchData will be broken for one day until PyTorch has updated it's nightly releases.

You find the DevEx problem that we have been suffering as torchdata is not off core.